client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -124,25 +124,31 @@ class PurchasesListing extends Component {
 		return null;
 	}
 
-	getExpirationInfo( purchase ) {
+	getExpirationInfoForPlan( plan ) {
+		// No expiration date for free plans.
+		if ( this.isFreePlan( plan ) ) {
+			return null;
+		}
+
+		const expiryMoment = plan.expiryDate ? this.props.moment( plan.expiryDate ) : null;
+
+		const renewMoment =
+			plan.autoRenew && plan.autoRenewDate ? this.props.moment( plan.autoRenewDate ) : null;
+
+		return <ProductExpiration expiryDateMoment={ expiryMoment } renewDateMoment={ renewMoment } />;
+	}
+
+	getExpirationInfoForPurchase( purchase ) {
 		// No expiration date for free plan or partner site.
 		if ( this.isFreePlan( purchase ) || isPartnerPurchase( purchase ) ) {
 			return null;
 		}
 
-		const subscribedMoment = purchase.subscribedDate
-			? this.props.moment( purchase.subscribedDate )
-			: null;
-
 		const expiryMoment = purchase.expiryDate ? this.props.moment( purchase.expiryDate ) : null;
 
-		return (
-			<ProductExpiration
-				expiryDateMoment={ expiryMoment }
-				purchaseDateMoment={ subscribedMoment }
-				isRefundable={ purchase.isRefundable }
-			/>
-		);
+		const renewMoment = purchase.renewDate ? this.props.moment( purchase.renewDate ) : null;
```

Comment:
Similar to my above comment, we may want to add an `isExpiring()` check here.

Reply:
Thanks. Added it here as well.